### PR TITLE
Add ipdb recipe

### PR DESCRIPTION
**Pull request by vamega**

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks @vamega.

**Comment by jakirkham** [comment]
@gotcha, just letting you know we have added `ipdb` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge ipdb`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/ipdb-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `ipdb`. 😄

**Comment by gotcha** [comment]
@jakirkham Happy that ``ipdb`` is useful enough for you to take time to add it to conda-forge. Sorry, but I do not know what conda-forge is. Where can I find an ELI5 ?

**Comment by jakirkham** [comment]
I think the ELI5 would just be the [website]( http://conda-forge.github.io/ ). The super short answer is this is a community-based package management system for the `conda` package manager. We have pretty close interactions with developers in the major SciPy projects, which helps us stay on top of recent releases and helps us resolve build issues quickly. Our infrastructure makes it very easy to add new packages and manage existing ones. Please let me know if you have any questions.
